ansys/dpf/core/field.py

```diff
@@ -87,32 +92,33 @@ class Field(_FieldBase):
     >>> fields_container = disp.outputs.fields_container()
     >>> field = fields_container[0]
     >>> field.data[2]
-    array([-0.00672665, -0.03213735,  0.00016716])
+    array([-0.00672665, -0.03213735, 0.00016716])
```

Comment:
```suggestion
    array([-0.00672665, -0.03213735,  0.00016716])
```
Another spot where we're changing the result and this breaks the build.